hydromt_wflow/wflow.py

```diff
@@ -3297,17 +3308,12 @@ def read_geoms(
         """
         Read static geometries and adds to ``geoms``.
 
-        Assumes that the `geom_fn` folder is located in the same folder as the static
-        input data (``input.path_static``). If not found uses assumes they are in
-        <root/geom_fm>.
+        Assumes that the `geom_fn` folder is located in the same folder as the root or
+        the folder where the config file is located (which is the root).
         """
         if not self._write:
             self._geoms = dict()  # fresh start in read-only mode
-        dir_default = join(self.root, "staticmaps.nc")
-        dir_mod = dirname(
-            self.get_config("input.path_static", abs_path=True, fallback=dir_default)
-        )
-        fns = glob.glob(join(dir_mod, geom_fn, "*.geojson"))
```

Comment:
I think I did that in case staticmaps path was for example ../../staticmaps. Then the user does not have to think too much about the staticgeoms path as you can assume it will most probably be in the same folder as staticmaps. The one thing I did not take into account and that leads to errors is that in the toml you can now specify a dir_input like you have a dir_output. I think it may be nicer to keep the current behavior and to also handle the dir_input for staticgeoms see #276 
What do you think? 
Else this means the user needs to manually specify the staticgeoms folder and to avoid making it too complex (ie can easily call read/write), I would then put the geom_fn as a __init__ argument of model.
So I would be okay it either 1 we keep the current code and add dir_input or 2) we keep your implementation but geom_fn can be defined in init.

Reply:
Well most of the time the staticmaps is in the same folder as the config file, i.e. the root directory, so I felt like that the addition of the staticmaps is a bit too much. Therefore, It made, at least to me, not a lot of sense to rely on the staticmaps. Maybe it's best to have a quick call regarding this.